src/components/User.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+
+const User = ({ name }) => <p>Name: <i>{name}</i></p>;
```

Comment:
```suggestion
function User({ name }) {
  return (
    <p>
      Name: <i>{name}</i>
    </p>
  );
};  
```
Though it's not a mistake, but for better readability, it's better to use the full syntax for functional components in React